### DISCUSSION TITLE
Organization: Implement different features to help cleanup files

### DISCUSSION THREAD

**Discussion by CWempe** [photoprism/photoprism#2350] [Ideas]
**Is your feature request related to a problem? Please describe.**

My library of photos gets bigger every day.
I guess many users can relate to this.

So I would like to clean up my files by deleting photos I do not need.

**Describe the solution you'd like**

It would be great if Photoprism could help me to clean up my library.

Google Photos and Google Files (Android Apps) have some features for this:

1. Categorize `memes`, `screenshots` and photos of `documents` wich usually are not wanted in a photo library.
2. Find duplicate files
3. List biggest files

I think photoprism already has some features needed for this, too.
What I am missing is:

1. Categorize `memes`, `screenshots` and photos of `documents`
2. Option to export a list of these files.  
  I am using the read-only mode, so I cannot just select the photos and delete them via Photoprism.  
  But I could use an exported list of the photos (including path) and delete them with a script.

**Describe alternatives you've considered**

Of cause there are already different tools for this work.
Like https://github.com/pauldreik/rdfind or https://dev.yorhel.nl/ncdu.

**Additional context**

**Comment by graciousgrey** [comment]
Thank you very much for your feedback!

I converted this into a discussion for now as it is not one clearly defined issue but rather a collection of ideas. 

PhotoPrism does already label photos as 'info' or 'document'.
Photos with these labels as well as non-photographic images such as memes should automatically appear in PhotoPrism's review section
 
https://docs.photoprism.app/user-guide/organize/review/

 Photos that are in review do not appear in your normal views. 

In addition we have the archive for photos that should not be shown in your normal views. You can select the photos you don't want to keep and move them to the archive. This won't change something on your filesystem.

For exporting filepaths we already have an open issue

#1518

For now you could move all photos you want to delete to the archive and then find the filepaths in the database. Archived photos have a deletedAt date set in the photos table.

Duplicates are detected and can be found in the database in the duplicates table.  Resolving duplicates using the Web UI is planned but also won't work with read-only mode.

https://docs.photoprism.app/user-guide/library/duplicates/

#1308 

Regarding the list with big files, this would be nice to have on a dashboard :)

**Reply by CWempe** [reply]
Thanks. I did not know about the review feature.

So I guess I will just have to wait for the two linked issued to be solved.
